tweepy/streaming.py

```diff
@@ -168,7 +170,7 @@ def read_line(self, sep='\n'):
                 return self._pop(loc + len(sep))
             else:
                 start = len(self._buffer)
-            self._buffer += self._stream.read(self._chunk_size)
+            self._buffer += self._stream.read(self._chunk_size).decode(self._encoding)
```

Comment:
That is a good point and one that I didn't really consider.  How do you feel about doing the decode as a part of the pop function.  I.e. keep everything internal to the ReadBuffer as bytes, but decode on returns.  The other option of course is to just keep everything in bytes and return bytes, and just offload the decoding to the person using ReadBuffer.  What is your opinion?